pytealext/evaluator/data/gen_op_costs.py

```diff
@@ -0,0 +1,47 @@
+import os
+import re
+
+here = os.path.dirname(os.path.abspath(__file__))
+
+# pattern = re.compile(r"^(?P<opcode>[a-z]+)(?P<arg>\d+)?$")
+pattern = re.compile(r"\"(?P<opcode>[^\"]+)\".+ (?P<cost_function>\b\w+\(.*\))}")
+
+matches: list[dict[str, str]] = []
+
+CHEAP_PATTERN = re.compile(
+    r"detDefault|detBranch|detSwitch|immediates|constants|field|typed|only|immKinded\(([^()]+)\)(?!costs)"
```

Comment:
dunno if `(?!costs)` works here since probably  immKinded would be called like this `immKinded.(...).costs`, or you had another intention